### DISCUSSION TITLE
HM-1500 ID: 141 Netz: Netzüberspannung

### DISCUSSION THREAD

**Discussion by ds2k5** [tbnobody/OpenDTU#1977] [General]
Hallo,
habe von meinen Eltern (aus Frankreich) einen Screenshot der Meldungen bekommen.
WR: HM-1500 / Firmware: 1.0.18 24.12.2021 / Booloader Version: 0.1.0 

Da steht: ID: 141 Netz: Netzüberspannung

Netzspannung ca. 238.7 Volt

Die Meldungen kamen wohl als sehr gute Sonneneinstrahlung vorhanden war.

WR Temperatur war so ca. bei 50 Grad

**Comment by MetaChuh** [comment]
hi @ds2k5

vielen dank für deine transparenten und hilfreichen informationen.
much appreciated.

welchen leitungsquerschnitt haben deine eltern vom inverter zum zählerkasten auf wieviel distanz ?
bzw. haben deine eltern im zählerkasten noch eine weitere messeinheit, die die volt am endpoint zum cross check nochmals messen kann, um die hm-1500 meldung auch extern zeitgleich messen zu können ?

thx & greetings,
metachuh

**Comment by Marc9485** [comment]
Den Hinweis von MetaChuh kann ich aus eigener Erfahrung bestätigen. 
Hatte von der UV-Garage zum WR1 ca.15m 3x1,5 und weiter WR1 und WR2 über in/out(AC) mit ca.5m 3x1,5 verbunden. Das lief im Winter erst mal wunderbar. Ab Februar hatte ich dann an WR2 an einigen "guten Tagen" das gleiche Problem und das dann mehrmals am Tag direkt hintereinander. An der UV die Spannung mit Duspol gemessen und da war die Spannung dann absolut okay. Über Home Assistant hab ich mir dann mal die Leistung von WR2 über den Tag angeschaut und jedes mal kurz vorm Peak(1.500Watt) brach die Leistung auf 0Watt ein das war dann die Netzüberspannung. Dann weiter die Spannung im Verlauf nachgeschaut und dort gesehen das die bis 252,9V hoch ging und kurz danach(wahrscheinlich dann bei 253V) wieder absackte. Da war dann die Netzüberspannung die ich mir selber eingebrockt hab. Hab dann ein 5x2,5 von der UV hoch gelegt und die WR's(1.-L1 & 2.-L2) jeweils mit 3x2,5 daran angeschlossen und seitdem nie mehr dieses Problem gehabt.
Auf 20m Länge mit 3.000 Watt max. Leistung hätte ich "doofer" Elektromaschinenbauer nicht mit gerechnet. Erfahrung macht klug...

**Comment by ds2k5** [comment]
@MetaChuh 
danke für die hilfe!

Also folgende Daten habe ich bekommen:

Netzspannung wohl bei 248 Volt gewessen heute.
Mehr Sonne als Gestern.
Temp heute war wohl bei 67 Grad bei ca. 1450 Watt
Aber keine Fehlermeldung heute

Kabel:

2,5 q  ca. 14m
danach ca. 6m 1,5 q
dann ca. 2m 1,5 q zum Wechselrichter


@Marc9485 

danke für die Hilfe und den Hinweis mit den 2,5q Leitungen, muss klären ob sich
das tauschen lässt ohne eine Baustelle (Schltize klopfen/fräsen) machen lässt.

**Comment by Marc9485** [comment]
Das die Spannung so hoch ist wird nicht am Netzbetreiber liegen. Wurde die Spannung an der UV oder am WR gemessen? 
Die 248V wurden aber auch wieder nur in einem Moment gemessen!?! Wenn du keine Möglichkeit hast die Spannung zu tracken; es gibt Multimeter mit Max./Min. Funktion. Damit köntest du dir den maximalen Spannungswert den das Messgerät an z.B. einem Tag gemessen hat anzeigen lassen. 

Habt ihr die Möglichkeit ein Kabel aussen her zu verlegen? "Quick and dirty" zum testen!?! Würde H05RN-F oder H07RN-F dafür nehmen. Kannst de ohne bedenken im freien benutzen, ist flexibel und somit einfach zu verlegen.

**Comment by ds2k5** [comment]
@Marc9485 

>Wurde die Spannung an der UV oder am WR gemessen?

Vom WR die Info via OpenDTU abgelesen

>Die 248V wurden aber auch wieder nur in einem Moment gemessen!?!

ja

>Habt ihr die Möglichkeit ein Kabel aussen her zu verlegen? "Quick and dirty" zum testen!?! 

Muss ich klären - ich bin nicht vor Ort lebe nicht in FRA

**Comment by ds2k5** [comment]
@schneeer 

Hallo,
laut der Info meiner Eltern hatten Sie nun ein paar Tage mit fast Sonne pur
und der Fehler ist nicht aufgetreten.

Mal abwarten...

@ALL
Danke euch für eure hilfe!

**Comment by ds2k5** [comment]
@schneeer 

danke
